packages/app/src/clickhouse.ts

```diff
@@ -144,7 +166,7 @@ export function useSessions(
             ],
             from: traceSource.from,
             dateRange,
-            where: `mapContains(${traceSource.resourceAttributesExpression}, 'rum.sessionId')`,
+            where: `mapContains(${traceSource.resourceAttributesExpression}, 'rum.sessionId') AND ${whereClauseSQL}`,
```

Comment:
It seems like we're better off if we use the `filters` param for the `mapContains` statement and then pass in the user input into `where` and that way we don't have to hack some weird stuff on top?

Reply:
I forgot about `filters` 😛  thanks for the tips. let me update the code